diff/windows/cimfs.go

```diff
@@ -50,9 +50,6 @@ func init() {
 				return nil, err
 			}
 
-			if !cimfs.IsCimFSSupported() {
-				return nil, fmt.Errorf("host windows version doesn't support CimFS")
```

Comment:
What if we return `ErrSkipPlugin` here?  That would keep the check centralized and the plugin from being shown as registered maybe? (https://github.com/containerd/containerd/blob/d75bc05ef350e1f3107ff72625be1e53c564cefb/snapshots/devmapper/plugin/plugin.go#L46)

Reply:
This works, thank you!